components/Breadcrumbs.tsx

```diff
@@ -110,12 +110,19 @@ const Breadcrumbs = ({
     <nav aria-label="breadcrumbs">
       <ol className={styles.breadcrumbs}>
         {homeBreadcrumb}
-        {breadcrumbs.map((breadcrumb, index) => (
-          <React.Fragment key={`${breadcrumb.title}-${index}`}>
-            <Breadcrumb breadcrumb={breadcrumb} index={index} />
-            {lowerCase(breadcrumb.title) === 'lessons' && <LessonsDropdown />}
-          </React.Fragment>
-        ))}
+        {breadcrumbs
+          .slice(
+            0,
+            breadcrumbs.findIndex(
```

Comment:
If you don't plan to use something, don't set it in the first place. Therefore, I think this logic should be handled in line 49.